docs/book/v1/intro.md

```diff
@@ -24,13 +24,25 @@ In order to implement the OAuth2 server, we first need to configure it. The
 first step is to generate new cryptographic keys. We need to execute the script
 `./vendor/bin/generate-oauth2-keys` in order to generate these keys.
 
+This script will store the keys in the application's `data` folder, if found:
+
 ```bash
 $ ./vendor/bin/generate-oauth2-keys
-```
 
-This script will store the keys in the application's `data` folder if found:
+This script is provided as a convenient way to generate keys for
+the OAuth2 server provider. You may choose instead to use an
+alternative method. For more information, see the install docs:
+https://oauth2.thephpleague.com/installation/
+
+Best available location for keys:
+./data
+You'll likely want to move them to a better location
+
+We'll put them in a subdirectory:
+./data/oauth
+
+Using 2048 bits to generate key of type RSA
```

Comment:
Where is this specified or does it happen automatically?

_Unfortunately, as a user, I don't know what I'm actually supposed to do in the section?_

Reply:
Output of the command 🙂